packages/blade/src/components/Typography/BaseText/__tests__/BaseText.native.test.tsx

```diff
@@ -0,0 +1,287 @@
+import React from 'react';
+import renderWithTheme from '../../../../_helpers/testing/renderWithTheme.native';
```

Comment:
Out of scope from this PR but we need aliases soon.

Reply:
yesss